src/datahike/api.cljc

```diff
@@ -85,12 +108,22 @@
   create-database
   dc/create-database)
 
+(s/fdef
+  delete-database
+  :args (s/alt :config spec/SConfig
+               :deprecated/uri string?
+               :nil (s/cat)) ; Picks up the default database
+  :ret (s/nilable map?)) ; TODO: should this be nilable?
```

Comment:
I think so, or should we something like a boolean?

Reply:
I would be for a boolean, especially for Datomic compatibility, but that's a different discussion.

At the moment though it seems to me like the return value of delete-database isn't even necessarily a map or nil but that this depends on the implementation of the store and basically could be anything. So either we are ok with that or define it more closely and check it in the integration tests. Anyway I think that should be a separate issue.

Reply:
I created discussion #599 about that so we can decide there.